plugin/src/main/java/net/aufdemrand/denizen/objects/dEntity.java

```diff
@@ -736,6 +734,18 @@ public dInventory getInventory() {
                 : dInventory.mirrorBukkitInventory(getBukkitInventory()) : null;
     }
 
+    public dList getTradeRecipes() {

```

Comment:
Why is this here and not in the property

Reply:
Leftover from the old PR, where I had three Trade-related properties instead of only one. Will fix.